Wikipedia/Code/WMFAppViewController.m

```diff
@@ -301,7 +301,7 @@ - (UINavigationController*)navigationControllerForTab:(WMFAppTabType)tab {
     return (UINavigationController*)[self.rootTabBarController viewControllers][tab];
 }
 
-- (UIViewController<WMFAnalyticsLogging>*)rootViewControllerForTab:(WMFAppTabType)tab {
+- (UIViewController<WMFAnalyticsViewNameProviding>*)rootViewControllerForTab:(WMFAppTabType)tab {
     return [[[self navigationControllerForTab:tab] viewControllers] firstObject];
```

Comment:
might want to add an assertion for `WMFAnalyticsViewNameProviding` conformance, otherwise VC's that don't conform will (silently?) fail to log